queue-health/graph/graph.py

```diff
@@ -85,6 +90,7 @@ def format_timedelta(delta):
 
 
 class Sampler(object):
+    mean = 0
```

Comment:
Right, this pattern only works for immutable types, which `int` is and
`list` is not.